lib/ansible/modules/windows/win_mssql_query.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Daniele Lazzari <lazzari@mailup.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a windows documentation stub. Actual code lives in the .ps1
+# file of the same name.
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_mssql_query
+version_added: '2.7'
+short_description: invokes a query on a MSSQL Server
+description:
+    - Invokes a query on a MSSQL Server. This module isn't idempotent, the query will be always executed.
+    - In example, if you run a query that creates a database C(CREATE DATABASE TestDb), the first time it will success
+    - the second time it will fail because the database already exists.
+requirements:
+  - Powershell module SQLPS or Powershell module SqlServer
+options:
+  server_instance:
+    description:
+      - Name of the sql server instance to query.
+    default: computername
+  server_instance_user:
+    description:
+      - Name of a sql user with sufficient privileges.
+  server_instance_password:
+    description:
+      - Password for server_instance_user
+  query:
+    description:
+      - Query to invoke.
+    required: yes
+  query_timeout:
+    description:
+      - Specifies the number of seconds before the queries time out.
+    default: 60
+  database:
+    description:
+      - database where the query has to be executed
+    default: master
+author:
+  - Daniele Lazzari
+notes:
+  - If the user that runs ansible has the appropriated permission on the db,
+  - you don't need to set C(server_instance_user) and C(server_instance_password).
```

Comment:
No need to put this in a new list entry, just put `you` in the newline where it is but remove `-`. You may want to add in that CredSSP, Kerb with cred delgation or become must be used for implicit auth to work.